src/main/assets/scss/main.scss

```diff
@@ -0,0 +1,5 @@
+$govuk-global-styles: true;
```

Comment:
@KiranVarma02 let's keep this turned on until such a time we find it problematic.